config-model/src/main/java/com/yahoo/vespa/model/container/ApplicationContainerCluster.java

```diff
@@ -50,11 +50,13 @@
     private ContainerModelEvaluation modelEvaluation;
 
     private Optional<TlsSecrets> tlsSecrets;
+    private final boolean enableGroupingSessionCache;
 
     public ApplicationContainerCluster(AbstractConfigProducer<?> parent, String subId, String name, DeployState deployState) {
         super(parent, subId, name, deployState);
 
         this.tlsSecrets = deployState.tlsSecrets();
+        this.enableGroupingSessionCache = deployState.getProperties().enableGroupingSessionCache();
```

Comment:
Redundant `this` (also on the line above).

Reply:
I agree that it is strictly not necessary; it's more about what code style a developer prefers.